Automattic/pocket-casts-ios
# New podcasts grid design

Fixes #

This PR updates the layout for the Podcast grid adding spacing and a new badge design.


https://github.com/Automattic/pocket-casts-ios/assets/651601/2244b8e9-df6e-451a-91ef-77d6d3ad6db2

| Large Grid | Small Grid | List |
| - | - | - |
| ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-19 at 13 09 00](https://github.com/Automattic/pocket-casts-ios/assets/651601/1ed64901-c58d-41e0-83ad-1d024ff853e6) | ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-19 at 13 09 03](https://github.com/Automattic/pocket-casts-ios/assets/651601/0bf4cdfe-f05a-4aad-9cbf-5afabb410c4e) |  ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-19 at 13 09 07](https://github.com/Automattic/pocket-casts-ios/assets/651601/38cbb190-e6bb-465e-a666-41eb27d0313f) |
| ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-19 at 13 08 36](https://github.com/Automattic/pocket-casts-ios/assets/651601/d97888d0-0730-483a-8a66-1e8021c2ef0b) | ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-19 at 13 08 33](https://github.com/Automattic/pocket-casts-ios/assets/651601/5b07d67a-110c-47b7-aea0-79690b6fa555) | ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-19 at 13 08 27](https://github.com/Automattic/pocket-casts-ios/assets/651601/8cedbcd7-91be-4390-8e82-b2cee1252872) |


## To test

 - Start the app
 - Open the Podcasts tab
 - Test the following
 - Check if the new layout is working correctly, by scrolling around on you podcast view
 - Open folders ( make a folder if you don't have one)
 - Tap on the top right more button, and switch between the different layout types: Small Grid, Large Grid, List. 
 - On each grid types check the different badge options: Off, Unfinished Episodes, Only Latest Episode
 - Check with different themes
 - Check on the iPad

## Checklist
